File: ingest/hello.php
<?php

$targetdir = '/Users/RLAS_Admin/Sites/ingest/uploads/';   
      // name of the directory where the files should be stored
// $targetfile = $targetdir.$_FILES['file']['name'];
$uploadedfile = $targetdir . basename($_FILES['file']['name']);

if (move_uploaded_file($_FILES['file']['tmp_name'], $uploadedfile)) {
    // file uploaded succeeded
} else { 
    // file upload failed
	echo "failure";

}
print_r($_FILES);
// $file = $_FILES["file"]["name"];
$user = $_POST["user"];

echo $user;

$command = escapeshellcmd('/usr/local/bin/python3 /Users/RLAS_Admin/Sites/ingest/hello.py' . $uploadedfile . $user);
// $command = escapeshellcmd('/usr/local/bin/python3 /Users/RLAS_Admin/Sites/ingest/help.py' . );
$output = shell_exec($command);
echo $output;

?>

File: ingest/hello.py
#!/usr/local/bin/python3

import sys, json, subprocess

myfile = sys.argv[1]

user = sys.argv[2]

# def process_file(something):
#     mediainfo = subprocess.call(['mediainfo', something])
#     # stuff = sys.stdout.write(mediainfo)
#     return mediainfo
    # return stuff

print("hello"+user)


# process_file(myfile)
# print(sys.stdout())

stuff = subprocess.run(['mediainfo', myfile], stdout=subprocess.PIPE)
print(stuff)